lib/types/uint8_array.ex

```diff
@@ -0,0 +1,27 @@
+defmodule Kadena.Types.Uint8Array do
```

Comment:
@santiagoboterokommit Uint8Array is similar to an Array where each item is an 8-bit (1 byte) unsigned integer. Uint8Arrays cannot change size after creation (this is not a problem since the elixir struct does not mutate).

That said, your structure could receive two types of data:

A list of positive integers.
A binary buffer that must be read from left to right every 1 byte and thus obtain the unsigned integer.
Note: be sure to handle the binary offset correctly.